zipline/finance/performance/position_tracker.py

```diff
@@ -449,11 +449,7 @@ def maybe_create_close_position_transaction(self, event):
             return None
 
         amount = self.positions.get(event.sid).amount
-
-        if 'price' in event:
-            price = event.price
-        else:
-            price = self._position_last_sale_prices[event.sid]
+        price = self._data_portal.get_current_price_data(event.sid, 'close')
```

Comment:
I have similar code on one of my branches, i.e. :+1:, will resolve any potential conflicts on my end.